reporting/report.go

```diff
@@ -75,7 +76,14 @@ func PrintHTML(dbsIn []string, res *resources.Resources) error {
 	for k := range dbs {
 		err = writeDB(dbs[k], wd, res)
 		if err != nil {
-			return err
+			if err.Error() == ("No results were found for " + dbs[k]) {
+				fmt.Println("[!] " + err.Error())
+				fmt.Println("    This might indicate that this database has been dropped, but the metadatabase hasn't been updated.")
+				fmt.Println("    To update the metadatabase and fix this problem, run:")
+				fmt.Println("    rita delete-database " + dbs[k])
```

Comment:
This might indicate that the database has been dropped. However, there could be other reasons. Either of these will also cause this issue:
* if the database exists but is empty
* if the database has data but it's from a previous, incompatible version of rita

Since there could be other reasons, I'm a little worried about recommending to delete the database to fix the problem. It should definitely work, but they may lose something they wanted to keep, especially if there are other situations this can come up that we don't know of. Do you think it would be better to just get rid of the recommendation and the `delete-database` command, or might it be better to expand further and say something like "You can either disregard this message or attempt to fix it by deleting the metadatabase entry. Warning: this will also delete any data in the specified database: <delete command here>"?

Reply:
I figure we could list a few of the potential reasons, and then give the recommendation with warning like you mentioned. Maybe something like:
"This might indicate that this database has been dropped without the metadatabase being updated, the database exists but is empty, or the data in the database is from an incompatible version of rita. You can either disregard this message, or attempt to fix it by deleting the metadatabase entry by running the following command:
rita delete-database <db_name>
WARNING: this will also delete any data in the specified database."

Reply:
It's getting wordy but it's not like you will see this often. We can go with that.